### PR TITLE
[ACR-1132] Add media codec feature

### PR DESCRIPTION
**Pull request by eunhaechoi**
- ACR-1132 : Add media codec feature

### CODE REVIEW

File: docs/application/native/tutorials/details/app-filtering.md [hunk 1]
```diff
@@ -186,6 +186,7 @@ package:
 | `http://tizen.org/feature/minicontrol`   | Specify this key, if the application requires the minicontrol feature. | 4.0   |
 | `http://tizen.org/feature/multi_point_touch.pinch_zoom` | Specify this key, if the application requires a pinch-zoom gesture feature. | 2.2.1 |
 | `http://tizen.org/feature/multi_point_touch.point_count` | Specify this key with a specific number (`int` type), if the application requires the minimum of specified number of simultaneous touches in a multi-point touch. | 2.2.1 |
+| `http://tizen.org/feature/multimedia.media_codec`     | Specify this key, if the application requires the media codec feature. | 4.0   |
```
Comment (mijin85cho) [inline docs/application/native/tutorials/details/app-filtering.md]: Question. We have feature tables for mobile and wearable. This feature is added to mobile feature table only. Doesn't the wearable support this feature?

**Comment by mijin85cho** [comment]
Hello, @eunhaechoi 
Would you please check my comment?

File: docs/application/native/tutorials/details/app-filtering.md (hunk 1 above)
Comment (eunhaechoi) [reply docs/application/native/tutorials/details/app-filtering.md]: It is already added in wearable section. I think the engineer who was in charge of modified the wearable section only at that time.

Comment (journey2w) [reply docs/application/native/tutorials/details/app-filtering.md]: I confirmed this change.

**Comment by journey2w** [comment]
@mijin85cho 
This change have to apply to TD because the feature started from Tizen 4.0.
I'd try to update TD but there is a scheduled revision in TD for Tizen 5.5. Let's discuss how to do next Monday.

**Pull request merged**